synapse/tests/test_model_gov_intl.py

```diff
@@ -0,0 +1,47 @@
+
+import synapse.lib.module as s_module
+
+from synapse.tests.common import SynTest
+
+testmodel = {
+
+    'types': (
+        ('fakem49', ('gov:intl:un:m49', {}), {'doc': 'A fake int type.'}),
+    ),
+
+    'forms': (
+        ('fakem49', {}, ()),
+    ),
+
+}
+
+class TestModule(s_module.CoreModule):
```

Comment:
Any reason to have a separate Module here?

Reply:
Agreed, you could probably just add your fake data model directly to the core.